UI/window-basic-settings.cpp

```diff
@@ -896,7 +897,10 @@ OBSBasicSettings::OBSBasicSettings(QWidget *parent)
 
 		ui->bindToIP->addItem(QT_UTF8(name), val);
 	}
-
+	// Add IP Version options
+	ui->ipVersion->addItem(QTStr("Basic.Settings.Advanced.Network.IPVersion.Both"), "0");
+	ui->ipVersion->addItem(QTStr("Basic.Settings.Advanced.Network.IPVersion.V4Only"), "4");
+	ui->ipVersion->addItem(QTStr("Basic.Settings.Advanced.Network.IPVersion.V6Only"), "6");
```

Comment:
I kind of want to suggest more descriptive names for the config options, since configuration files are editable by humans; but then not sure if "Both" vs "0" is better, or maybe "IPv4+IPv6" instead of "0"? Though I don't expect this to matter much

Reply:
Good call